src/Aquifer.Data/Migrations/20250224185220_AddVerseIdRangesTableType.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Aquifer.Data.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddVerseIdRangesTableType : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("""
+                CREATE TYPE [dbo].[VerseIdRanges] AS TABLE
```

Comment:
We could add a prefix or suffix to these (`UDTT` or `TableType` is common) if desired.